lib/api/tenure/v1/service.test.ts

```diff
@@ -126,7 +126,7 @@ describe("when editTenure is called", () => {
         assignedOfficer: {
           firstName: "Firstname",
           lastName: "Lastname",
-          email: "firstname.lastname@hackney.gov.uk",
```

Comment:
I've glanced over some BE PRs before this one, it didn't seem like `email` was getting removed.

Are we sure we want to exclude it from this test?

Reply:
Yeah I think you're right. I removed as we're not setting it and it's optional but wanted to keep it in be for a possible future implementation. So I think keeping it in the test, as it can be set, makes sense.